src/main/scala/devsearch.lookup/LookupMerger.scala

```diff
@@ -1,6 +1,6 @@
 package devsearch.lookup
 
-import akka.actor._
+import  akka.actor._
```

Comment:
Haha I have no idea how this could happen! :stuck_out_tongue: and i can even less imagine why i've pushed this...